src/snowflake/cli/plugins/git/manager.py

```diff
@@ -18,3 +18,15 @@ def show_files(self, repo_path: str) -> SnowflakeCursor:
     def fetch(self, repo_name: str) -> SnowflakeCursor:
         query = f"alter git repository {repo_name} fetch"
         return self._execute_query(query)
+
+    def create(
+        self, repo_name: str, api_integration: str, url: str, secret: str
+    ) -> SnowflakeCursor:
+        query = (
+            f"create git repository {repo_name}"
+            f" api_integration = {api_integration}"
+            f" origin = '{url}'"
+        )
```

Comment:
Let's use multiline string, it makes things a bit more readable

Reply:
done